inference-engine/thirdparty/clDNN/tests/test_cases/convolution_gpu_test.cpp

```diff
@@ -4883,6 +4883,22 @@ using TestParamType_general_convolution_gpu = ::testing::tuple<  int,    // 0 -
         std::string,    // 12 - Implementation name
         bool>;          // 13 - With bias
 
+using TestParamType_convolution_bfyx_padding_gpu = ::testing::tuple<
+    int,            // 0  - Input B size
+    int,            // 1  - Input F size
+    int,            // 2  - Input Y size
+    int,            // 3  - Input X size
+    int,            // 4  - Output F size
+    int,            // 5  - Kernel X size
+    int,            // 6  - Kernel Y size
+    int,            // 7  - Groups number
+    int,            // 8  - Stride
+    int,            // 9  - Output padding B
```

Comment:
I think we dont need to set output padding because this is not the deconvolution

Reply:
but seems that you are using these as input pad..

Reply:
Deleted output padding